x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/RealmsTests.java

```diff
@@ -481,17 +480,6 @@ public void testDisabledRealmsAreNotAdded() throws Exception {
         assertThat(realms.getUnlicensedRealms(), hasSize(orderToIndex.size()));
     }
 
-    public void testAuthcAuthzDisabled() throws Exception {
```

Comment:
Realms wouldn't be built if security is off

Reply:
Not a great deal, but a test in `SecurityTests` that `Security#createComponents` doesn't return no `Realms` would be nice.

Reply:
Added one in c079be4aa26ed685e30ce8ef1ff222781f244bd8. Not sure why `not(containsInAnyOrder(instanceOf(class)))` was not working though :/